ffmpeg/src/main/java/org/bytedeco/ffmpeg/presets/avfilter.java

```diff
@@ -33,7 +33,7 @@
  * @author Samuel Audet
  */
 @Properties(
-    inherit = {avformat.class, postproc.class, swresample.class, swscale.class},
+    inherit = {avformat.class, /*postproc.class,*/ swresample.class, swscale.class},
```

Comment:
This doesn't work if we leave postproc?

Reply:
I'm trying to revert and test building...

Reply:
It works well, so I'll push the commit.